src/pages/development/security/subresource-integrity.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Subresource Integrity
+description: Security feature that enables browsers to verify the resources they fetch.
+---
+
+# Subresource Integrity
+Subresource Integrity (SRI) is a security feature that enables browsers to verify that resources they fetch (for example, from a CDN) are delivered without unexpected manipulation. It works by allowing you to provide a cryptographic hash that a fetched resource must match.
+See [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to learn more about SRI.
+
+## Application Support
+As of version 2.4.7, Adobe Commerce and Magento Open Source add support for Subresource Integrity by providing integrity hashes for all javascript assets residing in the local filesystem. (This
+functionality is defined in the Magento_Csp module.)
+Please note that currently there is no Subresource Integrity support for custom remote javascript resources. Merchants need to implement and provide hashes for all remote javascript assets used in themes.
```

Comment:
I would recommend to remove the "implement" word from "Merchants need to implement and provide hashes..." to avoid making a false impression that implementation of some functionality beyond addition of an integrity attribute to remote scripts is required.